app/controllers/admin/users_controller.rb

```diff
@@ -45,6 +43,8 @@ def core_params
         location
         email_public
         education
+        password
+        password_confirmation
```

Comment:
why these?

Reply:
I wasn't sure if the `password` params should be removed, so I left them.

Reply:
i think you can remove them as well :) i'll ask someone else in the team to review this as well before we merge

Reply:
Ok, I will send the change.